starter_kits/Python3/hlt/entity.py

```diff
@@ -50,6 +50,16 @@ class Ship(Entity):
     """
     Ship class to house ship entities
     """
+    __ships = {}
+    
+    def __new__(cls, owner, id, position, halite):
```

Comment:
Can you add comment on what we are trying to do here?

Reply:
Since \_\_new\_\_ is not beginner friendly, I posted on the pull request with another possible solution (with inline comments)